backend/auth-service/src/main/kotlin/com/nosql/authservice/component/impl/DefaultUserComponent.kt

```diff
@@ -27,7 +26,7 @@ class DefaultUserComponent(
     private val log: Logger by logger()
 
     @Transactional
-    override suspend fun saveIfNotExists(user: UserEntity): UserEntity {
+    override suspend fun save(user: UserEntity): UserEntity {
         val operationDetails = "Save if not exists 'auth' record with login = ${user.login}"
```

Comment:
Change operation details to "Save 'auth' .."

Reply:
fixed